api/views/declaration/declaration.py

```diff
@@ -397,6 +400,80 @@ class OngoingDeclarationsListView(GenericDeclarationsListView):
     queryset = Declaration.objects.exclude(status=Declaration.DeclarationStatus.DRAFT).distinct()
 
 
+class DeclarationHyperlinkXLSXRenderer(XLSXRenderer):
+    """
+    Ceci est un hack pour ajouter des liens au noms de la déclaration. Cet issue
+    a été levé dans drf-excel : https://github.com/django-commons/drf-excel/issues/112
+    """
+
+    def render(self, data, media_type=None, renderer_context=None):
+        excel_data = super().render(data, media_type, renderer_context)
+        workbook = load_workbook(filename=BytesIO(excel_data))
+
+        # Ajouter le lien vers le résultat de la recherche dans le nom du produit
+        for sheet in workbook.worksheets:
+            for row in sheet.iter_rows(min_row=2):
+                name_cell = row[0]
+                hyperlink_data = data[name_cell.row - 2]["url_field"]
+                name_cell.value = hyperlink_data.display
+                name_cell.hyperlink = hyperlink_data.ref
+
+        # Sauvegarder en Bytes
+        output = BytesIO()
+        workbook.save(output)
+        return output.getvalue()
+
+
+class OngoingDeclarationsExcelView(XLSXFileMixin, CommonOngoingDeclarationView):
+    pagination_class = None
+    renderer_classes = [DeclarationHyperlinkXLSXRenderer]
+    serializer_class = ExcelExportDeclarationSerializer
+    filename = "declarations-resultats.xlsx"
+
+    # Format de l'entête du fichier Excel
+    column_header = {
+        "titles": [
+            "Nom du produit",
+            "Marque",
+            "Article",
+            "Statut",
+            "Nom de la compagnie",
```

Comment:
```suggestion
            "Nom de l'entreprise",
```

Pour être cohérent avec le wording sur les pages du site (par ex tableau de bord)